### PR TITLE
Bump rel 0.22/alt/nanterre

### PR DESCRIPTION
**Pull request by MoretS**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,52 +4,71 @@ source "https://rubygems.org"
...
+#gem "decidim-cookies", git: "https://github.com/OpenSourcePolitics/decidim-module_cookies", branch: "feature/orejime"
+#gem "decidim-combined_budgeting", git: "https://github.com/mainio/decidim-module-combined_budgeting"
+#gem "decidim-decidim_awesome", git: "https://github.com/Platoniq/decidim-module-decidim_awesome"
+#gem "decidim-comparative_stats", git: "https://github.com/Platoniq/decidim-module-comparative_stats"
+#gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications"
+#gem "decidim-homepage_interactive_map", git: "https://github.com/OpenSourcePolitics/decidim-module-homepage_interactive_map.git"
+#gem "decidim-initiatives_no_signature_allowed", git: "https://github.com/OpenSourcePolitics/decidim-module-initiatives_nosignature_allowed.git"
+#gem "decidim-navigation_maps", git: "https://github.com/Platoniq/decidim-module-navigation_maps"
+
+gem "decidim-socio_demographic_authorization_handler", git: "https://github.com/MoretS/decidim-module-socio_demographic_authorization_handler", branch: "release/0.22-stable"
```
Comment (Quentinchampenois) [inline Gemfile]: Hi @MoretS !

Should we not use the OpenSourcePolitics' repository for the gem `socio_demographic_authorization_handler` ?

File: Gemfile [hunk 2]
```diff
@@ -4,52 +4,71 @@ source "https://rubygems.org"
...
+
+gem "decidim-phone_authorization_handler", git: "https://github.com/MoretS/decidim-module_phone_authorization_handler", branch: "release/0.22-stable"
```
Comment (Quentinchampenois) [inline Gemfile]: Should we not use the OpenSourcePolitics' repository for the gem `phone_authorization_handler` ?

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]

**Comment by Quentinchampenois** [comment]
Seems good to me, thank you @MoretS !

File: Gemfile (hunk 1 above)
Comment (MoretS) [reply Gemfile]: Indeed ! Modification applied

File: Gemfile (hunk 2 above)
Comment (MoretS) [reply Gemfile]: Indeed ! Modification applied

**Pull request merged**